### PR TITLE
#162725248 Integrate TravisCI with readme badge

### PR DESCRIPTION
**Pull request by kdaglas**
#### What does this PR do?
This PR is to integrate TravisCI for the application and badges to the README.md

#### Description of Task to be completed?
Integrate TravisCI service for the application and when integration is done, ensure display of the build status using a build-badge in [ReadMe](https://github.com/andela/Ah-backend-xmen/blob/ch-integrate-travis-162725248/README.md)

#### How should this be manually tested?
- Navigate to this branch's [ReadMe](https://github.com/andela/Ah-backend-xmen/blob/ch-integrate-travis-162725248/README.md) file
- Click on the build badge which will take you to Travis report on travis-ci.org

#### What are the relevant pivotal tracker stories?
[#162725248](https://www.pivotaltracker.com/story/show/162725248)

#### Screenshots
image

### CODE REVIEW

**Review by MuhweziDeo — DISMISSED** [review]

**Review by calvinpete — DISMISSED** [review]
@kdaglas Looks good to me

**Review by malfahad — DISMISSED** [review]
LGTM

**Review by d-rita — DISMISSED** [review]
good stuff

File: README.md [hunk 1]
```diff
@@ -392,7 +394,3 @@ No additional parameters required
 ### Get Tags
 
 `GET /api/tags`
```
Comment (ja-odur) [inline README.md:396]: @kdaglas Add a newline at the end of this file.

Comment (kdaglas) [reply README.md:396]: I have added it

**Review by calvinpete — APPROVED** [review]
suggestion resolved, cool!

**Review by malfahad — APPROVED** [review]

**Review by armstrongsouljah — APPROVED** [review]
LGTM

**Review by ja-odur — APPROVED** [review]
LGTM

**Pull request merged**